frontend/src/features/documents/documentUpload/ComposedDocument.ts

```diff
@@ -0,0 +1,7 @@
+import { Api_Document } from 'models/api/Document';
+import { Api_Storage_DocumentMetadata } from 'models/api/DocumentStorage';
+
+export interface ComposedDocument {
+  mayanMetadata?: Api_Storage_DocumentMetadata[];
```

Comment:
nit: I like how we moved from `Mayan_DocumentMetadata` to `Api_Storage_DocumentMetadata` to avoid leaking mayan   details. Would you consider renaming this property name to match - ie "storageMetadata" ?

Reply:
For sure. That is something that could be done on the next pr that does something for documents given it would be another set of changes